tasks/agent.py

```diff
@@ -845,7 +845,7 @@ def _send_build_metrics(ctx, overall_duration):
                 }
             )
     dd_api_key = ctx.run(
-        f'{aws_cmd} ssm get-parameter --region us-east-1 --name ci.datadog-agent.datadog_api_key_org2 --with-decryption --query "Parameter.Value" --out text',
+        f'{aws_cmd} ssm get-parameter --region us-east-1 --name {os.environ.get("API_KEY_ORG2_SSM_NAME")} --with-decryption --query "Parameter.Value" --out text',
```

Comment:
Is it more readable like this ?
```suggestion
        f'{aws_cmd} ssm get-parameter --region us-east-1 --name {os.environ["API_KEY_ORG2_SSM_NAME"]} --with-decryption --query "Parameter.Value" --out text',
```

Reply:
Did the same comment on https://github.com/DataDog/datadog-agent/pull/23529#discussion_r1521846363 😄 
Looks like yours was missed though 🥲